components/member-list-item/member-list-item.module.scss

```diff
@@ -33,3 +33,8 @@
     width: 28px;
   }
 }
+
+.container:hover {
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+  transform: translateY(-10px);
+}
```

Comment:
Hovering indicates we can do something with the card, like clicking it our picking it. Is that what we expect to allow the user?

Reply:
Yeah, I mean after going through the basic structure, I observed on clicking the image we would be going to the details page.
Either that or I can just add box shadow to the card and remove the transition on hovering....
Open to suggestion

Reply:
So, we are doing something on clicking? Got it. Thanks for explaining @Harshi7016 :)